apps/website/src/pages/docs/radio.mdx

```diff
@@ -14,17 +14,21 @@ import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='input-radio' />
```

Comment:
Since removing this, we can safely remove the import on line 12:
`import Placeholder from '~/components/Placeholder.astro';`